pkg/controller/constraint/constraint_controller.go

```diff
@@ -303,8 +300,35 @@ func (r *ReconcileConstraint) Reconcile(ctx context.Context, request reconcile.R
 		status.Status.Errors = nil
 
 		if c, err := r.cfClient.GetConstraint(instance); err != nil || !reflect.DeepEqual(instance, c) {
+			err := util.ValidateEnforcementAction(enforcementAction, instance.Object)
+			if err != nil {
+				status.Status.Errors = append(status.Status.Errors, constraintstatusv1beta1.Error{Message: err.Error()})
```

Comment:
is there a test that validates this error is written to the constraint's status errors?

Reply:
Added the test [constrainttemplate_controller_test.go#L811](https://github.com/open-policy-agent/gatekeeper/pull/3321/files#diff-acfa8801c77d031956edb3d5c1078fe12346ff32dcda82fc3ce60a7ab9c013d3R811)